internal/aws/aws.go

```diff
@@ -0,0 +1,27 @@
+package aws
+
+import (
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+)
+
+func NewSession(r string) *session.Session {
```

Comment:
Being in the `internal` repository, all this is not exposed to the package but is only for internal consumption.